src/pantry.js

```diff
@@ -1,7 +1,97 @@
 class Pantry {
   constructor(userIngredients) {
     this.contents = userIngredients;
+    this.filtered
```

Comment:
I think these properties serve a purpose as part of the constructor for now, but we may find they exist better locally